package.json

```diff
@@ -49,6 +49,7 @@
         "d3-color": "^1.2.3",
         "d3-format": "^1.3.2",
         "d3-time-format": "^2.1.3",
+        "enzyme-to-json": "^3.3.4",
```

Comment:
Is this used anywhere?

Reply:
Now it is!  I missed the update in the jest config before, good catch.  Just pushed the fix.

`enzyme-to-json` translates the Enzyme shallow DOM rendering structure into JSON for use in Jest snapshot tests, so it looks much cleaner and can be easily diffed in (for instance) `LayerToolbar.spec.js.snap`